packages/nc-gui-v2/composables/useColumnCreateStore.ts

```diff
@@ -196,6 +196,11 @@ const [useProvideColumnCreateStore, useColumnCreateStore] = createInjectionState
           }
           await $api.dbTableColumn.create(meta.value.id as string, formState.value)
 
+          /** if LTAR column then force reload related table meta */
+          if (formState.value.uidt === UITypes.LinkToAnotherRecord && meta.value.id !== formState.value.childId) {
+            getMeta(formState.value.childId, true)
```

Comment:
Missing await?

Reply:
it's not required since we don't have to wait for it. It's updating the metadata of the related table.